jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/cpu/OverallCPULoadHandler.java

```diff
@@ -50,16 +58,37 @@ public void initializeMeter(Meter meter) {
             measurement -> {
               measurement.record(machine);
             });
+    meter
+        .gaugeBuilder(METRIC_NAME_MACHINE_MINUTE)
+        .setDescription(METRIC_DESCRIPTION_MACHINE_MINUTE)
+        .setUnit(UNIT_UTILIZATION)
+        .buildWithCallback(
+            measurement -> {
+              measurement.record(machineMinuteAverage);
+            });
   }
 
   @Override
   public void accept(RecordedEvent ev) {
+    // Synchronized to avoid races on machineSum
+    synchronized (this) {
+      if (ev.hasField(MACHINE_TOTAL)) {
+        machine = ev.getDouble(MACHINE_TOTAL);
+        // Remove oldest data point if queue is at capacity
+        if (machineMinuteQueue.size() == 60) {
+          machineSum -= machineMinuteQueue.poll();
+        }
+        // Add new data point
+        machineSum += machine;
+        machineMinuteQueue.add(machine);
+        // Compute new result
+        machineMinuteAverage = machineSum / machineMinuteQueue.size();
```

Comment:
This math isn't quite right. `machineSum` is a running total of all the `machine` measurements, and its being divided by the size of the deque which is always <= 60. 

You need to iterate through `machineMinuteAverage`, sum up the entries, and divide by the size.

Should do this in the callback on line 66 so you're only computing as needed as opposed to for each measurement.  And will need to synchronize so the deque isn't modified while computing the average.

Would be good to somehow breakout the pieces of code involved so it can be unit tested.